files/zh-cn/web/http/headers/warning/index.md

```diff
@@ -25,7 +25,7 @@ Warning: <warn-code> <warn-agent> <warn-text> [<warn-date>]
 
   - : 三位数字警告码。第一位数字表示 Warning 信息在验证之后是否需要从已存储的响应中删除。
 
-    - `1xx` 警告码描述了关于当前响应的新鲜度或者验证状态的警告信息，并且将会在验证之后被缓存服务器删除。
+    - `1xx` 警告码描述了关于当前响应的有效期或者验证状态的警告信息，并且将会在验证之后被缓存服务器删除。
```

Comment:
The English document does not match with the zh-CN document, I think there is a typo. Let's wait until mdn/content#28345 is resolved.

```suggestion
    - `1xx` 警告码描述了关于当前响应的有效期或者验证状态，并且将会在成功验证后被缓存服务器删除。
```